### PR TITLE
solved the pylint issue of  "no self argument"

### PR DESCRIPTION
**Pull request by Exoutia**
Issue number: #48855 
- [ ] xref https://github.com/pandas-dev/pandas/issues/48855 - one of many style checks, still needs work.
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [x] Added: just added the '@staticmethod' and '@classmethod' to solve it, pylint was acting up because of this.

### CODE REVIEW

File: pandas/tests/io/test_sql.py [hunk 1]
```diff
@@ -1613,6 +1613,7 @@ class _TestSQLAlchemy(SQLAlchemyMixIn, PandasSQLTest):
     flavor: str
 
     @pytest.fixture(autouse=True, scope="class")
+    @classmethod
```
Comment (MarcoGorelli) [inline pandas/tests/io/test_sql.py:1616]: I think you need to swap these two for pytest collection to work
```suggestion
    @classmethod
    @pytest.fixture(autouse=True, scope="class")
```

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]

**Review by MarcoGorelli — APPROVED** [review]
Looks good to me pending green

**Comment by Exoutia** [comment]
> Looks good to me pending green
> 
>  [ 68747470733a2f2f692e666c756666792e63632f344872646d4777374d484c33726348334a306c434242473778686c786e5773372e676966 ](https://camo.githubusercontent.com/0ef58abdbf19d92d1e6d43b55172c0a2afa644306616fc4e10b6f39fa2fab60d/68747470733a2f2f692e666c756666792e63632f344872646d4777374d484c33726348334a306c434242473778686c786e5773372e676966) [ ](https://camo.githubusercontent.com/0ef58abdbf19d92d1e6d43b55172c0a2afa644306616fc4e10b6f39fa2fab60d/68747470733a2f2f692e666c756666792e63632f344872646d4777374d484c33726348334a306c434242473778686c786e5773372e676966)

🤣🤣Loved the cat 😺😺

**Comment by twoertwein** [comment]
Thanks @Exoutia!

**Pull request merged**